Library/Formula/llvm.rb

```diff
@@ -128,6 +129,8 @@ def install
     (buildpath/"tools/lld").install resource("lld") if build.with? "lld"
     (buildpath/"tools/lldb").install resource("lldb") if build.with? "lldb"
 
+    commit_source
```

Comment:
Because we added more source code via a resource.   We dont it thrown away with the obect files